docs/how-tos/managing-yarn-dependencies.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Managing Yarn Dependencies
+---
+
+## On Your Machine
+
+FxA uses [Yarn](https://yarnpkg.com/) to manage its Node dependencies. Follow the [Development Setup guide](./development-setup) for how to install Yarn and the `yarn` command on your machine.
+
+:::note
+Once upon a time FxA used [`npm`](https://docs.npmjs.com/cli/v7/commands/npm) to manage its dependencies, which means you may come across documentation or commands that still reference its usage. In most cases you should be able to freely replace the keyword `npm` with `yarn`, but it's worth learning about [the differences](https://www.sitepoint.com/yarn-vs-npm/#comparingnpmandyarncommands) between the two CLIs.
+:::
+
+We have a _lot_ of dependencies, and they're constantly updating, so be sure to occasionally run `yarn install` to stay up-to-date.
```

Comment:
Could be worth noting that if you don't have (or don't want) `FXA_AUTO_INSTALL=1` set, a warning will be displayed letting you know if `yarn.lock` differs, which I personally prefer 🤷‍♀️

Reply:
I'll add that